doc/release-notes.rst

```diff
@@ -171,8 +193,19 @@ Major Changes from Kraken
     - Ubuntu 16.04 Xenial (x86_64 and aarch64)
     - Ubuntu 14.04 Trusty (x86_64)
 
-  * *CLI changes*:
+  * A first release of Ceph for FreeBSD is available which contains a full set
+    of features, other than Bluestore. It will run everything needed to build a
+    storage cluster. For clients, all access methods are available, albeit
+    CephFS is only accessible through a Fuse implementation. RBD images can be
+    mounted on FreeBSD systems through rbd-ggate
 
+    Ceph versions are released through the regular FreeBSD ports and packages
```

Comment:
eliminating this newline will (i think) make the blank lines in teh rendered output go away.